detect/master_changer.go

```diff
@@ -0,0 +1,119 @@
+package detect
+
+import (
+	"encoding/binary"
+	"net"
+	"strconv"
+
+	"github.com/mesos/mesos-go/detector"
+	mesos "github.com/mesos/mesos-go/mesosproto"
+
+	"github.com/mesosphere/mesos-dns/logging"
+)
+
+func init() {
```

Comment:
`main()` included (and still includes) a manual call to `logging.SetupLogs()`. This `init()` is misplaced, you are right. I'll move it to `master_changer_test.go`.